src/UI/AppLayout.jsx

```diff
@@ -16,11 +11,13 @@ function AppLayout() {
   };
 
   return (
-    <StyledAppLayout>
-      <Header toggleModal={toggleModal} />
+    <div className="box-border w-full">
```

Comment:
Just a heads up, `div`in this setting has border box and width 100% by default. It can change though if the parent is `flex`.

But in this case, it should be safe to remove these two classes